internal/kittehs/lists.go

```diff
@@ -36,6 +36,18 @@ func ContainedIn[T comparable](xs ...T) func(T) bool {
 	}
 }
 
+func NotContainedIn[T comparable](xs ...T) func(T) bool {
```

Comment:
Why do we need this?

Reply:
I agree, we don't need this

if you insist, probably better to return !ContainedIn which is already implemented and just negates the result (for small arrays, the performance is neglectable and it's more readable and less code to maintain)